### PR TITLE
Fix migrations worker

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?

(Provide a description of what this PR does and why it's needed.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/views/install/compose.phtml [hunk 1]
```diff
@@ -502,6 +502,37 @@ services:
...
+      - _APP_REDIS_PASS
+      - _APP_DB_HOST
+      - _APP_DB_PORT
+      - _APP_DB_SCHEMA
+      - _APP_DB_USER
+      - _APP_DB_PASS
+      - _APP_LOGGING_PROVIDER
+      - _APP_LOGGING_CONFIG
+      - _APP_MIGRATIONS_FIREBASE_CLIENT_ID
+      - _APP_MIGRATIONS_FIREBASE_CLIENT_SECRET
```
Comment (stnguyen90) [inline app/views/install/compose.phtml:534]: should these 2 be included in the `appwrite` container too so we can check the vars with `docker compose exec appwrite vars`?

**Review by stnguyen90 — APPROVED** [review]
Looks good. just 1 question

File: app/views/install/compose.phtml (hunk 1 above)
Comment (Meldiron) [reply app/views/install/compose.phtml:534]: Firebase secrets are on appwrite container

**Comment by Meldiron** [comment]
File dif comparing `docker-compose.yml` to `compose.phtml`: https://www.diffchecker.com/sASIAiLt/

File: app/views/install/compose.phtml [hunk 2]
```diff
@@ -351,6 +351,9 @@ services:
     restart: unless-stopped
     networks:
       - appwrite
+	  volumes:
+      - appwrite-functions:/storage/functions:rw
+      - appwrite-builds:/storage/builds:rw
```
Comment (stnguyen90) [inline app/views/install/compose.phtml:356]: Incorrect indentation...and I'm surprised this isn't conflicting with https://github.com/appwrite/appwrite/pull/6133/files

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Comment by christyjacob4** [comment]
@Meldiron can you look into the indentation of the file ?

File: app/views/install/compose.phtml (hunk 2 above)
Comment (Meldiron) [reply app/views/install/compose.phtml:356]: Weird, it shows okay on my end
CleanShot 2023-09-06 at 10 13 12@2x

Comment (Meldiron) [reply app/views/install/compose.phtml]: Rewrote it, good now. some space/tab issues

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**